src/components/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { shallow } from "enzyme";
```

Comment:
Now that we have `testing-library` it would be good to write new tests using it instead of enzyme.

There is an example in SummaryButton component:
https://github.com/canonical-web-and-design/react-components/blob/main/src/components/SummaryButton/SummaryButton.test.tsx

Could you update these tests so they don't use enzyme?